lib/simplifyExpression/collectAndCombineSearch/divideLikeTerms.js

```diff
@@ -0,0 +1,178 @@
+const arithmeticSearch = require('../arithmeticSearch');
```

Comment:
how much of this code is copied from multiplying and how much is new? ideally very little code should be copied